docs/assignment1.md

```diff
@@ -1 +1,53 @@
-Reverted this file
+## Introduction									
+Author(s): `Noé Rousset`
+
+This document introduces the Fantasy Soccer Project, addressing the specifications and requirements of the system and the libraries to be used to complete the project.
+ 
+This project closely follows the standard format of fantasy football games, in that a user assembles imaginary teams, whose performance is statistically evaluated. Unlike the original format, however, for this game, the data, as well as the schedule, are both provided by the player. 
+With the players added by the user, they can form a team of 11 players consisting of 1 goalkeeper, 4 defenders, 4 midfielders and 2 forwards. The score of each player is also added by the user, and the score of a team is the cumulative score of all the players. 
+After that team is formed, a schedule is set by the user and a tournament is played with other random teams. The teams play 1 on 1 matches in a single-elimination format[1], where the team with the highest total score wins. After a tournament is over, the user can form a new team, start a tournament again with the current team  or close the game.
+Possible improvements to this would be to use data for the players found online instead of the user adding it manually, and changing the way matches are played. In this upgraded version, each player would have the probability of committing a certain action, or an event happening based on their stats. For example a player with a strong attack would have a higher probability of scoring a goal. The score of a team would thus change between each game depending on their performance, with different actions [2]  earning a certain amount of points. 
+
+[1] https://en.wikipedia.org/wiki/Single-elimination_tournament
+
+[2] https://en.wikipedia.org/wiki/Fantasy_football_(association)#Points_scoring
+
```

Comment:
**[OVERALL]** pretty good description, some choices you made such as one user team and randomization of other teams make your design more interesting in a single player offline sense (++). There is however no distinction between the role of team manager and league manager, without this precision I can only assume that the user is not only a team manager but also a league manager (w/ actions such as setting the schedule and uploading player scores). Think about this further during your design phase